client/components/segmented-control/README.md

```diff
@@ -108,6 +108,10 @@ Boolean representing the selected visual state. `selected={ true }` creates a bl
 
 Optional URL to navigate to when option is clicked.
 
+`title`
+
+Optional title to show when hovering over segmented control options.
```

Comment:
Might be good to mention that the child content of the item will be displayed unless this is specified.